src/main/java/com/example/zawody/controller/AppController.java

```diff
@@ -0,0 +1,34 @@
+package com.example.zawody.controller;
+
+
+import com.example.zawody.model.DTO;
+import com.example.zawody.model.Person;
+import com.example.zawody.repositories.PersonRepository;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+import java.net.URI;
+
+@RestController
+public class AppController {
+
+    PersonRepository personRepository;
+
+    public AppController(PersonRepository personRepository) {
+        this.personRepository = personRepository;
+    }
+
+    @RequestMapping(value = "/posted",method = RequestMethod.POST,consumes = MediaType.APPLICATION_JSON_VALUE)
+    @CrossOrigin("http://localhost:8080")
+    public ResponseEntity<Void>save(@RequestBody Person person) {
```

Comment:
This should be ResultDTO that contains Person and Competition data. Then you can pass the ResultDTO to the service, and service should build Person and Competition entities based on the dto's data.